build_tester/builders/host_builder.py

```diff
@@ -0,0 +1,55 @@
+import os
+import traceback
+from collections import namedtuple
```

Comment:
Looks like imports are not in alphabet order.

Reply:
I tried to keep consistency with other files